backend/database/schemas.py

```
@@ -350,7 +370,7 @@ class UserLogin(BaseModel):
 
 class UserLoginResponse(BaseModel):
     access_token: str
-    token_type: str = "bearer"
+    token_type: str = "cookie"  # HttpOnly cookie authentication
```

Ruff (0.13.1)

Possible hardcoded password assigned to: "token_type"

(S105)